### DISCUSSION TITLE
No nodes spawn :([BUG]

### DISCUSSION THREAD

**Discussion by erzoelmatador** [ErinaSugino/Starbound-SBR-Mod#94] [Trouble Shooting] [answered]
## Describe the bug
>When using critterbang! or anything, to try to fuck them, nothing spawns

<br>

## Log File

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15072213/starbound.log)

<br>

## Additional context
i have frackin universe installed, don't know if its a problem, I also tried on my gamepass version and nothing happened either
<br>
![image](https://github.com/ErinaSugino/Starbound-SBR-Mod/assets/54250541/d3cf28b8-17c3-4ff1-ade7-63a6f0812c5d)

**Comment by ErinaSugino** [comment, marked as answer]
You have old Sexbound installed, which is exploding your game.
